xml/deployment_prep_zseries_prep_i.xml

```diff
@@ -54,44 +54,36 @@
    </para>
   </tip>
   <note>
-   <title>No Installation <quote>From Hard Disk</quote></title>
+   <title>No Installation from Hard Disk</title>
    <para>
-    It is not possible to install from hard disk by putting the content of the
```

Comment:
Here I would prefer the simpler form 'to install' (instead of 'to perform an installation'). KISS.